packages/resource-deployment/templates/dashboard.template.json

```diff
@@ -1035,7 +1035,7 @@
                                         },
                                         {
                                             "name": "Query",
-                                            "value": "customEvents\n| where name == \"FunctionalTest\"     and customDimensions.logSource == \"TestRun\"     and customDimensions.runId == toscalar(         customEvents\n| where name == \"FunctionalTest\" and customDimensions.testContainer == \"FinalizerTestGroup\"\n| top 1 by timestamp desc nulls last\n| project tostring(customDimensions.runId)      )\n| where customDimensions.testContainer != \"FinalizerTestGroup\"\n| project testGroup = customDimensions.testContainer, testName = customDimensions.testName, result = customDimensions.result, releaseId = customDimensions.releaseId, timestamp\n| sort by tostring(testGroup)\n"
+                                            "value": "let functionalTestEvents = customEvents\r\n| where name == \"FunctionalTest\";\r\nlet lastCompletedRun = functionalTestEvents\r\n| where customDimensions.testContainer == \"FinalizerTestGroup\"\r\n| top 1 by timestamp desc nulls last\r\n| project runId = customDimensions.runId, releaseId = tostring(customDimensions.releaseId), key=1;\r\nlet lastStartedRun = functionalTestEvents\r\n| top 1 by timestamp desc nulls last\r\n| project runId = customDimensions.runId, releaseId = tostring(customDimensions.releaseId), key=1;\r\nlet runId = toscalar(lastCompletedRun\r\n| join kind=fullouter lastStartedRun on key\r\n| project value = tostring(iff(isnotnull(releaseId) and releaseId == releaseId1, runId, runId1)));\r\nlet relevantTestEvents = functionalTestEvents\r\n| where customDimensions.runId == runId;\r\nlet completeTestRuns = relevantTestEvents\r\n| where customDimensions.logSource == \"TestRun\"\r\n| project testContainerName = tostring(customDimensions.testContainer), testName = tostring(customDimensions.testName), result = tostring(customDimensions.result), releaseId = customDimensions.releaseId, timestamp;\r\nlet completeTestContainers = relevantTestEvents\r\n| where customDimensions.logSource == \"TestContainer\"\r\n| project testContainerName = tostring(customDimensions.testContainer);\r\nlet testContainerNames = functionalTestEvents\r\n| where customDimensions.source == \"BeginTestSuite\"\r\n| top 1 by timestamp\r\n| project testContainerName = split(tostring(customDimensions.functionalTestGroups), \",\")\r\n| mv-expand testContainerName\r\n| project testContainerName = tostring(testContainerName);\r\ntestContainerNames\r\n| project testContainerName, testName = \"Additional tests\", result = \"incomplete\"\r\n| join kind=leftanti completeTestContainers on testContainerName\r\n| union completeTestRuns\r\n| where testContainerName != \"FinalizerTestGroup\"\r\n| sort by testContainerName\r\n"
```

Comment:
if we merge on container then we cannot see a test failing name but a fixed message `Additional tests` only. Is there a way to see test that is actually failing?

Reply:
We'll see the pass/fail results for individual tests, plus incomplete "additional tests" if the whole test container hasn't yet finished.

Reply:
Can you please post some screenshots with example. The above should have both group/test failures listed but only group is marked as failed

Reply:
Posted a screenshot below